src/pages/index.js

```diff
@@ -32,6 +32,7 @@ const features = [
   {
     title: "APIs & Clients",
     imageUrl: "img/apisclients.png",
+    // TODO: update this URL to apis-tools once that page exists in production
```

Comment:
These are the 5 "feature" blocks on the homepage. They are currently hardcoded to production. I'm going to try updating these to be relative paths as part of this PR. 

(Otherwise I'll have to leave this one pointed at a URL that is incorrect, and rely on a redirect correcting it when the PR is deployed.)